README.md

```diff
@@ -64,12 +64,12 @@ All resources share the same interface when making API calls. Use `.create` to c
 Applicants are the object upon which Onfido checks are performed.
 
 ```java
-onfido.applicant.create(requestBody)                    # => Creates an applicant
-onfido.applicant.update("applicant_id", requestBody)    # => Updates an applicant
-onfido.applicant.destroy("applicant_id")                # => Schedule an applicant for deletion
-onfido.applicant.restore("applicant_id")                # => Restore an applicant scheduled for deletion
-onfido.applicant.find("applicant_id")                   # => Finds a single applicant
-onfido.applicant.list()                                 # => Returns all applicants
+onfido.applicant.create(<REQUEST_BODY_HERE>)                         # => Creates an applicant
```

Comment:
Very minor: Could we put inverted commas around these? i.e. `"<REQUEST_BODY_HERE>"`, `"<APPLICANT_ID_HERE">` etc.

Sorry to be so pedantic!

Reply:
Actually in hindsight, maybe not with `"<REQUEST_BODY_HERE>"`... bit of an awkward one. If you and @ciaran16 are happy without the inverted commas that's fine.